Tests/SwiftLintFrameworkTests/DisableAllTests.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  DisableAllTests.swift
+//  SwiftLint
+//
+//  Created by Frederick Pietschmann on 3/11/18.
+//  Copyright © 2018 Realm. All rights reserved.
+//
+
+import Foundation
+@testable import SwiftLintFramework
+import XCTest
+
+class DisableAllTests: XCTestCase {
+     /// Example violation (identifer_name). Could be replaced with any other single violation.
+    private let violatingPhrase = "let r = 0\n"
```

Comment:
can you add other violation triggered by other rule in the example? I want to make sure that it disables several ones, just not one.

Reply:
Done